test/EndToEnd/ProjectBuildTests.cs

```diff
@@ -392,7 +392,7 @@ private static string DetectExpectedDefaultFramework(string template = "")
             string[] runtimeFolders = Directory.GetDirectories(Path.Combine(dotnetFolder, "shared", "Microsoft.NETCore.App"));
 
             int latestMajorVersion = runtimeFolders.Select(folder => int.Parse(Path.GetFileName(folder).Split('.').First())).Max();
-            if (latestMajorVersion == 7)
+            if (latestMajorVersion == 8)
             {
                 return $"net{latestMajorVersion}.0";
```

Comment:
This code will likely need a fallback for any inbox templates that are still at 7.0 (once we figure out why the templates aren't being found). See the changes here: https://github.com/dotnet/installer/commit/62a57d56a299072cedf05b60385cd6292ae4a1e3